### PR TITLE
feat(embedded): add documentation on embeddable engine

### PR DESCRIPTION
**Pull request by baptistemesta**
There is a new page with mention that it is a lab feature

Closes [BR-121](https://bonitasoft.atlassian.net/browse/BR-121)

### CODE REVIEW

**Comment by educhastenier** [comment]
retest this please

File: md/taxonomy.md [hunk 1]
```diff
@@ -142,6 +142,7 @@
       * [Install a Bonita cluster](install-a-bonita-bpm-cluster.md)
       * [Cluster administration](cluster-administration.md)
     * [Multi-tenancy and tenant configuration](multi-tenancy-and-tenant-configuration.md)
+  * [Embed engine (labs)](embed-engine.md)
```
Comment (educhastenier) [inline md/taxonomy.md]: labs / LABS / lab / LABS?

Comment (baptistemesta) [reply md/taxonomy.md]:
```suggestion
  * [Embed engine - LAB](embed-engine.md)
```

Comment (baptistemesta) [reply md/taxonomy.md]: must change everywhere with LAB

**Comment by educhastenier** [comment]
retest this please

**Comment by educhastenier** [comment]
retest this please

**Pull request merged**